### DISCUSSION TITLE
unignore / WEEK_PROGRAM_POINTER

### DISCUSSION THREAD

**Discussion by factory42** [SukramJ/aiohomematic#1670] [Q&A] [answered]
Irgendwie habe ich den Gebrauch von „unignore“ noch nicht verstanden.
Meine Datei sieht so aus:

WINDOW_STATE
ACTIVE_PROFILE
WEEK_PROGRAM_POINTER

Die ersten beiden Zeilen funktionieren für meine Homematic-IP Thermostate. Die letzte Zeile sollten für meine alten Homematic Thermostate sein, zeigt aber keine Wirkung.

Wie kann ich das aktuelle Thermostat-Programm in eine HA-Entität bringen?

Danke!
Matthias

**Comment by SukramJ** [comment, marked as answer]
> Irgendwie habe ich den Gebrauch von „unignore“ noch nicht verstanden.

Das kann ich bestätigen.
WEEK_PROGRAM_POINTER ist aus dem Paramset Master, und das funktioniert so nicht.

Am Besten auf die neue Version wechseln, und über die UI konfigurieren, dann passiert sowas nicht.
Bitte Release-Notes lesen.

**Reply by factory42** [reply]
> Am Besten auf die neue Version wechseln, und über die UI konfigurieren, dann passiert sowas nicht.

Funktioniert. Danke!
